app/src/main/java/com/pysiak/simple/clock/ui/TabAdapter.java

```diff
@@ -1,38 +1,63 @@
 package com.pysiak.simple.clock.ui;
 
-import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
 import android.support.v4.app.FragmentManager;
 import android.support.v4.app.FragmentStatePagerAdapter;
 
 import java.util.ArrayList;
 import java.util.List;
 
+/**
```

Comment:
Keep right sequence of items in javadoc - description at the top, @author next to it

Reply:
Also description is not informative.... Just change it to something like:
Provides fragments and tab titles to view pager associated with tab layout